core/src/wallet/tezos/TezosLikeAccount.h

```diff
@@ -154,6 +154,9 @@ namespace ledger {
             void incrementOptimisticCounter(std::shared_ptr<TezosLikeTransactionApi> tx, const std::shared_ptr<BigInt>& explorerCounter);     
 
             void saveOptimisticCounter(const std::shared_ptr<BigInt>& counter, const std::string& txHash);
+
+            void getCurrentDelegate(const std::shared_ptr<std::string> & callback);
+            FuturePtr<std::string> getCurrentDelegate();
```

Comment:
The function cannot be called by users of the libcore if not defined in the interface.